crates/oxc_linter/src/snapshots/no_typos.snap

```diff
@@ -0,0 +1,50 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_typos
+---
+  ⚠ eslint-plugin-next(no-typos): getStaticpaths may be a typo. Did you mean getStaticPaths?
+   ╭─[no_typos.tsx:4:1]
+ 4 │                 }
+ 5 │                 export const getStaticpaths = async () => {};
```

Comment:
can we point to `getStaticpaths` instead?

Reply:
rather than the whole export stmt

Reply:
fixed it!